GitUI/CommandsDialogs/FormStash.cs

```diff
@@ -116,7 +117,12 @@ private void LoadGitItemStatuses(IList<GitItemStatus> gitItemStatuses)
             Stashed.GitItemStatuses = gitItemStatuses;
             Loading.Visible = false;
             Stashes.Enabled = true;
-            toolStripButton1.Enabled = true;
+            refreshToolStripButton.Enabled = true;
+        }
+
+        private void ResizeStashedDropdown()
```

Comment:
Rename to ResizeStashWidth()?
It is primarily the Width that is changed (which is indirectly the default size for the dropdown).